addons/mail/static/src/models/composer/composer.js

```diff
@@ -567,6 +537,12 @@ function factory(dependencies) {
         isDoFocus: attr({
             default: false,
         }),
+        /**
+         * If true composer will log a note, else a message will be posted
```

Comment:
dot at the end

PS: here I don't like the term "message" technically (both are messages), but this is what people say functionlly.